lib/Client.php

```diff
@@ -76,7 +76,9 @@ public function __construct(string $apiKey, array $options = [], ?HttpClient $ht
             $options['ssl'] ?? true,
             (int) ($options['maximum_backoff_duration'] ?? 10000),
             false,
-            $options["debug"] ?? false
+            $options["debug"] ?? false,
+            null,
```

Comment:
what's this null for

Reply:
https://github.com/PostHog/posthog-php/blob/master/lib/HttpClient.php#L49